src/ds/d400/d400-device.cpp

```diff
@@ -552,7 +552,7 @@ namespace librealsense
         auto raw_sensor = get_raw_depth_sensor();
         _pid = group.uvc_devices.front().pid;
 
-        _is_mipi_device = val_in_range(_pid, { RS457_PID, RS430_GMSL_PID, RS415_GMSL_PID });
+        _is_mipi_device = val_in_range(_pid, { RS457_PID, RS430_GMSL_PID, RS415_GMSL_PID, RS401_GMSL_PID });
```

Comment:
Maybe we should have a static array of those PID's and not hard code it twice?
Also on line 506?
Better to have a single place there we device if it's a MIPI device.
Thoughts?